<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Employee</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="buttonSubmit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>410</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>submit</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>171</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="buttonMaintainCard">
       <property name="text">
        <string>maintain timecard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCreateReport">
       <property name="text">
        <string>create employee report</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonMaintainOrder">
       <property name="text">
        <string>maintain purchase order</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonPayment">
       <property name="text">
        <string>select a payment method</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>828</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
